dvc/output/base.py

```diff
@@ -470,6 +471,48 @@ def move(self, out):
         if self.scheme == "local" and self.use_scm_ignore:
             self.repo.scm.ignore(self.fspath)
 
+    def transfer(
```

Comment:
For the record: this might eventually turn into `commit` or get somehow unified with it, but at least having `transfer` here is great for now.